pl_spv_forecasts/src/services/meteo_data/visualcrossing-api/fetch_data.py

```diff
@@ -0,0 +1,49 @@
+from pandas import DataFrame, Series, concat, to_datetime
+from datetime import date
+import requests
+from dataclasses import dataclass
+from utils import columns as selected_columns
+
+
+@dataclass
```

Comment:
read about Pydantic - it could be better for requests and fetching data validation

https://docs.pydantic.dev/latest/